generator/app/index.js

```diff
@@ -0,0 +1,112 @@
+"use strict";
+
+module.exports = {
+  description: "Create app and route files",
+  prompts: [
+    {
+      type: "input",
+      name: "name",
+      message: "Name of the app",
+      validate: (value) => {
+        if (/.+/.test(value)) {
+          return true;
+        }
+        return "name is required";
+      },
+    },
+    {
+      type: "input",
+      name: "routeName",
+      message: "Name of the route",
+      validate: (value) => {
+        if (/.+/.test(value)) {
+          return true;
+        }
+        return "route is required";
+      },
+      description: "This is used in router.js",
+    },
+    {
+      type: "confirm",
+      name: "InjectRepository",
+      default: false,
+      message: "Would you inject repository in the methods?",
+    },
+    {
+      type: "input",
+      name: "repositoryName",
+      message: "Name of existing repository.",
+      description: "Only needed if previous answer was in affirmitive?",
+    },
+    {
+      type: "input",
+      name: "domainName",
+      message: "Name of primary domain object class.",
+    },
+  ],
+  actions: (data) => {
+    // Generate index.js and index.test.js
+
+    let appRelativePath = "src/app/{{camelCase name}}/";
+    let routeRelativePath = "src/interfaces/http/modules/{{camelCase name}}/";
```

Comment:
> Filename convention - `camelCase` should never be used. This leaves `snake_case` and `kebab-case`, I prefer `snake_case` for file.

No worries if you cannot edit it man I will do it within this week 👀

Reply:
ok

Reply:
Question do we need to generate the domain object as well ? that would imply asking for some initial properties like sequelize does for it's models?

Reply:
Good question, we can just have the domain object without properties for now (probably add some dummy property). 

Anyway, our goal is to remove some boilerplate from their end so I think this is enough.